src/core/AutoRest/AutoRest.json

```diff
@@ -35,6 +35,9 @@
     },
     "AzureResourceSchema": {
       "type": "AzureResourceSchemaCodeGenerator, AutoRest.AzureResourceSchema"
+    },
+    "Go": {
```

Comment:
Whats the difference in generic and Azure.go? We only use it for generating Azure Go SDK ARM code.